### ISSUE TITLE
Inconsistent behaviour for keyboard focus on wordcloud

### ISSUE THREAD

**Issue by franklinefrancis** [highcharts/highcharts#15447]
#### Expected behaviour
Keyboard navigation works consistently for wordcloud similar to that of charts. That is, if container is set to receive focus first, it receives focus on first 'Tab' key press. Focus shifts to data point only upon next 'Tab' key press.

#### Actual behaviour
Upon navigating to wordcloud using 'Tab' key, focus is grabbed by data point even if container is set to receive focus first. This is seen in v8.2.0 but seems to be working fine at least partially in the latest version (note that data points never receive focus).

**Broken Version**
![8 2 0-Broken-Version](https://user-images.githubusercontent.com/1227574/113712418-b9a18b00-9703-11eb-922d-134194a32dd6.gif)

**Partially Working Version**
![9 0 1-Partially-Working-Version](https://user-images.githubusercontent.com/1227574/113712517-d9d14a00-9703-11eb-8cdc-4d72fcd5e657.gif)

#### Live demo with steps to reproduce
Broken: https://jsfiddle.net/franklinefrancis/b7420k6w/
Partially Working: https://jsfiddle.net/franklinefrancis/nkdoq048/

#### Product version
8.2.0

#### Affected browser(s)
All browsers

**Comment by KacperMadej** [comment]
Thank you for reporting the bug.

fyi: a11y @oysteinmoseng 

_Internal note:_
v8.2.0 was focusing on points only and the current version is focusing only on the chart - cannot tab through points.

**Comment by oysteinmoseng** [comment]
@franklinefrancis The v8.2.0 behavior where the points are immediately focused is the intended behavior, and what we would consider best practice for accessibility. This is so that only interactive elements are keyboard focused, and to avoid unnecessary tab-points for keyboard only users.

I have however added functionality to make the container itself a tab-stop, in cases where you might want that. This should be done by adding `container` first in the [keyboardNavigation.order](https://api.highcharts.com/highcharts/accessibility.keyboardNavigation.order) list, and not by modifying the tabindex of the container in HTML.

Example: https://jsfiddle.net/oysteinmoseng/849sgtoj/

**Comment by franklinefrancis** [comment]
Thanks for the fix @oysteinmoseng, appreciate it. Can it be backported to 8.2.0?

The default behaviour to immediately focus on a data point is debatable, if the container is set to receive focus with tabindex attribute. We do have some keyboard operations that need to be supported at container level, such as the following:

![KB-Menu](https://user-images.githubusercontent.com/1227574/114020995-d2d64300-988d-11eb-944a-fea7da763303.png)

**Comment by franklinefrancis** [comment]
Thanks for your inputs @oysteinmoseng, will give CustomComponents a try. We have some challenges in adding wrapper div though.
